code-challenges.js

```diff
@@ -58,6 +104,24 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+const returnLast = (Array,number) => {
+    /* The purpose of this code is to input an array and a wanted value. Then return the last index of the wanted value. Note with our example below, there are two identical values. So we want the object that appears last in the array. 
+    Eg: const numberOfConnections = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+        const givenValue1 = 42
+        Expected output: 7
+
+        const givenValue2 = 10
+        Expected output: 8
+    */
+    let res = ''
+    for (let i = 0; i < Array.length; i++) {
+        if (Array[i] === number){
+            res = i
+        }
+    }
+    return res
+} 
```

Comment:
Same things here as my previous comments. Pseudo code and camelCase. Notice how `Array` is a different color than the other parameter `number`. That's because in JavaScript, capital A `Array` is a reserved keyword. To avoid unforeseen errors, make sure you're using camelCase for those parameters!